src/main/java/org/broadinstitute/hellbender/utils/samples/Sample.java

```diff
@@ -46,9 +58,18 @@ public Sex getSex() {
         return gender;
     }
 
+    public Affection getAffection() {
+        return affection;
+    }
+
     @Override
     public String toString() {
-        return String.format("Sample %s fam=%s dad=%s mom=%s gender=%s", ID, familyID, paternalID, maternalID, gender);
+        return String.format("Sample %s fam=%s dad=%s mom=%s gender=%s affection=%s", ID, familyID, paternalID, maternalID, gender, affection);
+    }
+
+    @Override
+    public int compareTo(final Sample sample) {
+        return ID.compareTo(sample.getID());
```

Comment:
`compareTo()` and `equals()` are inconsistent (that is, `a.compareTo(b)` == 0 does not imply `a.equals(b)`) -- either add the standard disclaimer "This class has a natural ordering inconsistent with equals()` or fix this.